corehq/apps/users/models.py

```diff
@@ -2708,6 +2710,10 @@ def get_id(self):
     def is_active(self):
         return True
 
+    @property
```

Comment:
I added this so that a unit test would work, but I want to call attention to this change to make sure it's alright.

Reply:
This should be fine, though I don't think the tests should be using `AnonymousCouchUser`s (more about that above)